app/views/AttachmentView.tsx

```diff
@@ -109,6 +110,7 @@ const AttachmentView = (): React.ReactElement => {
 
 	const setHeader = () => {
 		let { title } = attachment;
+		const isImageUrlBase64 = attachment.image_url && isImageBase64(attachment.image_url);
```

Comment:
And here you could just

```suggestion
		const isImageUrlBase64 = isImageBase64(attachment.image_url);
```